uint8 drone_id
float32 velocity